tasks/system_probe.py

```diff
@@ -282,14 +283,33 @@ def kitchen_prepare(ctx):
 
 
 @task
-def kitchen_test(ctx):
+def kitchen_test(ctx, target=None):
     """
     Run tests (locally) using chef kitchen against an array of different platforms.
     * Make sure to run `inv -e system-probe.kitchen-prepare` using the agent-development VM;
     * Then we recommend to run `inv -e system-probe.kitchen-test` directly from your (macOS) machine;
     """
+
+    # Retrieve a list of all available vagrant images
+    images = {}
+    with open(os.path.join(KITCHEN_DIR, "platforms.json"), 'r') as f:
```

Comment:
I don't think we need to do this check, because `inv kitchen.genconfig` will do it for us.

Reply:
I find useful to print a list of available images to folks invoking the task.
Also we're building there a map of image => to platform, so you can use only a single argument as opposed to `osversion` and `platform`.

Reply:
Could you move the image printing logic to `kitchen.py`? It would be useful to everyone, not just folks using the system-probe kitchen task. I know it already prints available lists of platforms and providers, so it should be easy to add.

Reply:
It's not clear to me how this would fit in the existing `kitchen.py` code.
If you have anything specific in mind please feel free to commit to this branch.
In terms of ergonomics I'd like to have a single argument to this function and eliminate the redundant `platform` one.

Reply:
Oh, I see what you are doing. You want to accept only `ubuntu-16-04` and have it infer `ubuntu`. I suppose this is fine then.